ovirt_imageio/client/_options.py

```diff
@@ -69,7 +72,18 @@ def __new__(cls, name=None, args=(), config=False, required=False,
             help (str): Help message to show in the online help.
         """
         return tuple.__new__(
-            cls, (name, args, config, required, type, default, help))
+            cls, (name, args, config, required, action, type, default, help))
+
+    def make_kwargs(self):
```

Comment:
Maybe implement this as read only property so we can do:

    cmd.add_argument(*options.args, **option.kwargs)